zyte_common_items/pages/search_request_template.py

```diff
@@ -0,0 +1,11 @@
+from web_poet import ItemPage
+
+from zyte_common_items.items import SearchRequestTemplate, SearchRequestTemplateMetadata
+
+from .mixins import HasMetadata
+
+
+class SearchRequestTemplatePage(
+    ItemPage[SearchRequestTemplate], HasMetadata[SearchRequestTemplateMetadata]
+):
+    pass
```

Comment:
This module misses Base and Auto page objects

Reply:
On purpose. Should we provide them?